roles/sosreport/tasks/main.yml

```diff
@@ -0,0 +1,82 @@
+---
+- name: sosreport - install required packages
```

Comment:
``sosreport -`` as prefix should be removed everywhere

Reply:
Done